services/frontend/src/components/profileForms/careerManagementForm/CareerManagementForm.jsx

```diff
@@ -81,6 +83,24 @@ const PersonalGrowthForm = ({ formType }) => {
     );
   };
 
+  /**
+   * Get saved Qualified Pools
+   *
+   * get saved Qualified Pools from profile
+   */
+  const getSavedQualifiedPools = () => {
+    const ll = {
+      qualifiedPools: profileInfo.qualifiedPools.map((i) => ({
+        id: i.id,
+        classificationId: i.classification.id,
+        jobTitle: i.jobTitle,
+        selectionProcessNumber: i.selectionProcessNumber,
+        jobPosterLink: i.jobPosterLink,
+      })),
+    };
+    if (profileInfo.qualifiedPools) setSavedQualifiedPools(ll.qualifiedPools);
```

Comment:
Maybe remove the need of having an intermidiary variable `ll`

```suggestion
    if (profileInfo.qualifiedPools) {
      setSavedQualifiedPools({
      qualifiedPools: profileInfo.qualifiedPools.map((i) => ({
        id: i.id,
        classificationId: i.classification.id,
        jobTitle: i.jobTitle,
        selectionProcessNumber: i.selectionProcessNumber,
        jobPosterLink: i.jobPosterLink,
      })),
    })
    }
```